.gitignore

```diff
@@ -26,6 +26,7 @@ yarn-error.log*
 **/.idea/*
 /.env.secrets
 
+.env
```

Comment:
we should also exclude the env sample as we don't need them anymore right?

Reply:
Yes you will need it, since when you run the app locally it looks for the urls in **process.env**

Reply:
so we need now a process.env in our locals with these values which no longer is deployed

Reply:
We always needed the process.env, when we make calls to the api from the frontend we get the base url from there. And yes we are no longer including the .env in the repo since it should not. That is why we include the  **env.sample** so each person knows what they need to include in ther .env locally